backend/app.py

```diff
@@ -172,6 +177,47 @@ class CsetsUpdate(BaseModel):
     row_index_data_map: Dict[int, Dict[str, Any]] = {}
 
 
+@app.get("/concept-set-names")
+@app.get("/datasets/csets/names")
+@app.get("/jq-cset-names")
+def cset_names() -> Union[Dict, List]:
+    """Get concept set names"""
+    return csets_read(field_filter=['conceptSetNameOMOP'])
+
+
+@app.get("/datasets/csets")
+def csets_read(field_filter: Union[List[str], None] = Query(default=None)) -> Union[Dict, List]:
+    """Get concept sets
+
+    field_filter: If present, the data returned will only contain these fields. Example: Passing `conceptSetNameOMOP` as
+    the only field_filter for OMOPConceptSet will return a string list of concept set names.
+
+    Resources: jq docs: https://stedolan.github.io/jq/manual/ , jq python api doc: https://github.com/mwilliamson/jq.py
+    """
+    if field_filter:
+        if len(field_filter) > 1:
+            d = {'error': 'Currently only 1 field_filter is allowed.'}
+        else:
+            # TODO: Need to replace this with Python API. Otherwise, deployment will be harder and must global
+            #  installation of JQ.
+            query = f"jq '.[] | .{field_filter[0]}' {CSETS_JSON_PATH}"
+            output, err = jq_wrapper(query)
+            if err:
+                return {'error': str('error')}
+            d = output.split('\n')
+            # x[1:-1]: What gets returned is a \n-delimited string of names, where each name is formatted
+            # as '"{NAME}"',so we need to remove the extra set of quotations.
+            # TODO: This operation likely needs to be done in a variety of cases, but I don't know JQ well enough to
+            #  anticipate all of the situations where this might arise. - joeflack4 2022/08/24
+            # todo: Is this List[str] preferable to List[Dict,? e.g. [{'conceptSetNameOMOP': 'HEART FAILURE'}, ...]?
+            d = [x[1:-1] for x in d]
```

Comment:
FYI:
> `# x[1:-1]: What gets returned is a \n-delimited string of names, where each name is formatted as '"{NAME}"',so we need to remove the extra set of quotations.`
> `# TODO: This operation likely needs to be done in a variety of cases, but I don't know JQ well enough to anticipate all of the situations where this might arise. - joeflack4 2022/08/24`